app/Console/Commands/GenerateTestData.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Race;
+use App\Models\User;
+use App\Models\Profile;
+use App\Models\Application;
+use App\Models\Recommendation;
+use Illuminate\Console\Command;
+
+class GenerateTestData extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $signature = 'generate:testdata {amount=1} {type=all}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Make some test data';
+
+    public function fire()
+    {
+        $number = (int) $this->argument('amount');
+        $type = ($this->argument('type') == 'all') ? null : $this->argument('type');
+
+        // plain user
+        if (! $type || $type == 'user') {
+            $user = factory(User::class, $number)->create()->each(function ($user) {
+                dump('plain user - ' . $user->id);
```

Comment:
Let's use [Artisan's output methods](https://laravel.com/docs/5.5/artisan#writing-output) to write these!

```
$user = factory(User::class, $number)->create()->each(function ($user) {
    $this->line('Created a plain user with ID: ' . $user->id);
});
```

Reply:
oh yeah duh never updated that whoops! thanks!